packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -595,18 +753,17 @@ String _generateSemanticsTree(DebugSemanticsDumpOrder childOrder) {
 /// rendering layer directly. If you are writing to a higher-level
 /// library, such as the Flutter Widgets library, then you would use
 /// that layer's binding (see [WidgetsFlutterBinding]).
+///
+/// The [RenderingFlutterBinding] can manages multiple render trees. Each render
```

Comment:
```suggestion
/// The [RenderingFlutterBinding] manages multiple render trees. Each render
```

Also, it feels like these two paragraphs are duplicates of the ones in the deprecated `renderView` accessor. I suppose if it is deprecated it doesn't matter so much, but maybe a macro would be good?

Reply:
I ended up not using a macro here because when the deprecation period of `renderView` runs out, I want to be able to straight-up just delete it without fuzzing around with these docs again.

Reply:
Makes sense.